sdk/storage/azure-storage-blob/tests/test_container_async.py

```diff
@@ -1250,40 +1250,76 @@ def test_delete_blobs_simple(self):
         assert response[2].status_code == 202
 
     @record
-    def test_delete_blobs_snapshot(self):
+    def test_delete_blobs_simple_no_raise(self):
+        if TestMode.need_recording_file(self.test_mode):
+            return
         loop = asyncio.get_event_loop()
-        loop.run_until_complete(self._test_delete_blobs_snapshot())
+        loop.run_until_complete(self._test_delete_blobs_simple())
 
-    async def _test_delete_blobs_snapshot(self):
+    async def _test_delete_blobs_simple_no_raise(self):
         # Arrange
         container = await self._create_container()
         data = b'hello world'
 
         try:
-            blob1_client = container.get_blob_client('blob1')
-            await blob1_client.upload_blob(data)
-            await blob1_client.create_snapshot()
+            await container.get_blob_client('blob1').upload_blob(data)
             await container.get_blob_client('blob2').upload_blob(data)
             await container.get_blob_client('blob3').upload_blob(data)
         except:
             pass
-        blobs = await _to_list(container.list_blobs(include='snapshots'))
-        assert len(blobs) == 4  # 3 blobs + 1 snapshot
 
         # Act
         response = await _to_list(await container.delete_blobs(
             'blob1',
             'blob2',
             'blob3',
-            delete_snapshots='only'
+            raise_on_any_failure=False
         ))
         assert len(response) == 3
         assert response[0].status_code == 202
-        assert response[1].status_code == 404  # There was no snapshot
-        assert response[2].status_code == 404  # There was no snapshot
+        assert response[1].status_code == 202
+        assert response[2].status_code == 202
+
+    @record
+    def test_delete_blobs_snapshot(self):
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(self._test_delete_blobs_snapshot())
+
+    async def _test_delete_blobs_snapshot(self):
+        # Arrange
+        container = await self._create_container()
+        data = b'hello world'
 
+        try:
+            blob1_client = container.get_blob_client('blob1')
+            await blob1_client.upload_blob(data)
+            await blob1_client.create_snapshot()
+            await container.get_blob_client('blob2').upload_blob(data)
+            await container.get_blob_client('blob3').upload_blob(data)
+        except:
+            pass
         blobs = await _to_list(container.list_blobs(include='snapshots'))
-        assert len(blobs) == 3  # 3 blobs
+        assert len(blobs) == 4  # 3 blobs + 1 snapshot
```

Comment:
Why? There is no reasons this assert changes

Reply:
hmm, looks like something went wrong in rebasing...